app/routes/index.js

```diff
@@ -54,4 +54,7 @@ module.exports = function (app) {
     app.post("/api/collection", Controllers.CollectionController.addCollection);
     app.put("/api/collection", Controllers.CollectionController.updateCollection);
     app.delete("/api/collection", Controllers.CollectionController.deleteCollection);
```

Comment:
We don't need these API endponts. We don't save collections in our backend database.
They are in smart contract.